riscv/src/runtime.rs

```diff
@@ -197,25 +225,25 @@ impl Runtime {
             [
                 format!(
                     "instr affine_256 link ~> {};",
-                    instr_link("arith.affine_256", 3, 24, 16) // will use registers 3..27
+                    instr_link("arith.affine_256", 0, 24, 16) // will use registers 3..27
                 ),
                 format!(
                     "instr ec_add link ~> {};",
-                    instr_link("arith.ec_add", 4, 32, 16) // will use registers 4..36
+                    instr_link("arith.ec_add", 0, 32, 16) // will use registers 4..36
                 ),
                 format!(
                     "instr ec_double link ~> {};",
-                    instr_link("arith.ec_double", 2, 16, 16) // will use registers 2..18
+                    instr_link("arith.ec_double", 0, 16, 16) // will use registers 2..18
                 ),
                 format!(
                     "instr mod_256 link ~> {};",
-                    instr_link("arith.mod_256", 3, 24, 8) // will use registers 3..27
+                    instr_link("arith.mod_256", 0, 24, 8) // will use registers 3..27
```

Comment:
The comments are outdated. Also, I think the `start_idx` parameter of `instr_link` can be removed, we always pass `0` now.